.github/workflows/build-bootc-images.yaml

```
@@ -269,3 +258,56 @@ jobs:
+          dnf -y install podman
+          tee "/etc/containers/registries.d/${{ env.OCI_REGISTRY }}.yaml" > /dev/null <<EOF
+            docker:
+              ${{ env.OCI_REGISTRY }}:
+                use-sigstore-attachments: true
+          EOF
+
+      - name: Log in to OCI registry
+        run: podman login -u '${{ secrets.QUAY_PUBLISHER_USERNAME }}' -p '${{ secrets.QUAY_PUBLISHER_PASSWORD }}' ${{ env.OCI_REGISTRY }}
+
+      - name: Set env vars
+        run: |
```

actionlint (1.7.4)

property "format" is not defined in object type {image: any}

(expression)